<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Background-color: rgb(0,0,0);color:rgb(255,255,255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_plot">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2f4f4f;
color: #FFFFFF;s</string>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rst">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2f4f4f;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ntc_sch.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>10</y>
      <width>177</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lbl_Vref">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>Vref:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_Rntc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>Rntc:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_Rv">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>Rv:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="line_Vref">
         <property name="styleSheet">
          <string notr="true">Background-color: rgb(255,255,255);color:rgb(0,0,0)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_Rntc">
         <property name="styleSheet">
          <string notr="true">Background-color: rgb(255,255,255);color:rgb(0,0,0)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_Rv">
         <property name="styleSheet">
          <string notr="true">Background-color: rgb(255,255,255);color:rgb(0,0,0)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>10</y>
      <width>197</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lbl_Bn">
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>Bn:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_Rp">
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>Rp:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_adc">
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>ADC(bit)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="line_Bn">
         <property name="styleSheet">
          <string notr="true">Background-color: rgb(255,255,255);color:rgb(0,0,0)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_Rp">
         <property name="styleSheet">
          <string notr="true">Background-color: rgb(255,255,255);color:rgb(0,0,0)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_adc">
         <property name="styleSheet">
          <string notr="true">Background-color: rgb(255,255,255);color:rgb(0,0,0)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_ix">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Num. of interpolating points(ix):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_const">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#2f4f4f;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>NTC Table</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_ix">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: rgb(255,255,255);color:rgb(0,0,0)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
